pkgs/stdenv/generic/setup.sh

```diff
@@ -986,7 +986,18 @@ patchPhase() {
 
 
 fixLibtool() {
-    sed -i -e 's^eval sys_lib_.*search_path=.*^^' "$1"
+    local search_path
+    for flag in $NIX_LDFLAGS; do
```

Comment:
Do we need to account for the role here? If `libtool` is ever used in `depsBuildBuild` we need to look into a different variable or does that never happen?

Reply:
I'm not 100%, but I don't think so — remember, libtool isn't a dependency we're providing (and when we are providing it, all that it's doing is generating the script this applies to), so I don't think it makes sense to think of it in any role but host.  What do you think
?

Reply:
Right, the question then becomes: Are there other search paths (for `build` and `target`, maybe) we should take care of as well?

Reply:
You mean other paths libtool looks at?  I don't think so — when linking a proogram, it looks at two paths specific to the program being linked, sys_lib_search_path (what we're using here), and `LD_LIBRARY_PATH`.

Reply:
Hmm, I see. What's matched by `sys_lib_.+search_path`?

Reply:
`sys_lib_dlsearch_path`.  I'm not sure what it does, but it's not used in the same way as the other paths.